src/Result/LightningInvoice.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+use BTCPayServer\Util\PreciseNumber;
+
+class LightningInvoice extends AbstractResult
+{
+    public function getId(): ?string
+    {
+        return $this->getData()['id'];
+    }
+
+    public function getStatus(): ?string
+    {
+        return $this->getData()['status'];
+    }
+
+    public function getBolt11(): ?string
```

Comment:
not sure with this one, your screenshot says it returns NULL but according to API docs it can't be NULL, but as this is not fixed yet we need to keep it like you have it here now unfortunately

Reply:
Yep I'll take the nulls out once btcpay fixes it.